docs/src/start/dylib.md

```diff
@@ -40,13 +56,15 @@ abi_stable = "0.11.1"
 sys-info = "0.9.1"
```

Comment:
I think abi_stable is actually not needed.

Also, need to double check but steel-core might require adding "dylibs" feature.

Reply:
Good catch, that is right, the dylib feature is necessary. 

I also think you're right that abi stable isn't necessary for this example. More advanced usages  might, but I also need to just reexport Abi stable from steel itself so downstream consumers don't need to specify Abi stable as a dependency

Reply:
So, it was needed by a macro.

Reply:
Dang, if that is the case then we definitely need to re export abi_stable and have the macro reference those directly

Reply:
Do you by chance know which macro needed it?

Reply:
I think it was `declare_module`.

Can't check for sure as I won't have a dev environment for at least a week.